src/rust/engine/options/src/parse.rs

```diff
@@ -43,10 +43,10 @@ peg::parser! {
         }
 
         rule unquoted_string() -> String
-            = s:(non_escaped_character() / escaped_character())+ { s.into_iter().collect() }
+            = s:(escaped_character() / non_escaped_character())+ { s.into_iter().collect() }
 
         rule non_escaped_character() -> char
-            = !"\\" c:$([_]) { c.chars().next().unwrap() }
```

Comment:
A backslash *can* be a non-escaped character, if the subsequent character isn't an escape sequence.  

Note that this requires us to change the order of escaped_character() / non_escaped_character() above and below, to ensure that we first check for escaping and then let the backslash through if it isn't an escape.